testsuite/utils/src/main/java/org/keycloak/testsuite/TotpGenerator.java

```diff
@@ -34,7 +34,7 @@ public static void main(String[] args) throws IOException {
 
         Timer timer = new Timer();
         TotpTask task = null;
-
+        String secret = null;
```

Comment:
Minor: This line seems to be redundant and can be removed. Class `TotpGenerator` likely doesn't need to be changed in this PR